tasktiger/worker.py

```diff
@@ -52,7 +52,7 @@ def __init__(
         single_worker_queues=None,
         max_workers_per_queue=None,
         store_tracebacks=None,
-    ):
+    ) -> None:
```

Comment:
mypy needs return types on all of these but doesn't care about arg types?

Reply:
I only added the return types so that `mypy` would do at least basic checks of some functions.
We'll need to add more typing later and then gradually make the config stricter.